core/app/modules/import-export/directories/root.php

```diff
@@ -96,7 +96,7 @@ protected function import( array $import_settings ) {
 
 		$new_settings = array_replace_recursive( $old_settings, $new_settings );
 
-		$kit->save( [ 'settings' => $new_settings ] );
+		Plugin::$instance->kits_manager->create_new_kit( $import_settings['title'], $new_settings );
```

Comment:
You calling to kits_manager instance twice. 
`Plugin::$instance->kits_manager`
Look at line 77.

Reply:
Plugin::$instance->kits_manager it's an object, not a function, it's ok to use it twice. also, if I use it once I would have to create a new variable and use it twice.